rfcs/004-deal-scraper.md

```diff
@@ -0,0 +1,42 @@
+# RFC: On-chain deal scraper
+
+## Abstract
+
+This RFC proposes the design and functionality of an application to map `piece_cids` to their respective owners, identified by [Peer ID][1]s, in the Polka-Storage ecosystem.
```

Comment:
I'm wondering, who needs that from the business perspective at this point.
When a Storage Client is making a deal, they know with whom, so they *should know* where their file is stored.
The only thing they need is to resolve PeerID to a Multiaddress and then connect to the SP, download the file and that's it.

Not sure who the user of it would be.